gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -230,13 +230,14 @@ const postChangeUnissuedFacility = async (req, res) => {
     });
 
     const facilityEndDateRedirectUri = `/gef/application-details/${dealId}/unissued-facilities/${facilityId}/facility-end-date`;
+    const bankReviewDateRedirectUri = `/gef/application-details/${dealId}/unissued-facilities/${facilityId}/bank-review-date`;
     const unissuedFacilitiesRedirectUri = `/gef/application-details/${dealId}/unissued-facilities`;
 
-    if (isUsingFacilityEndDate === true) {
-      return res.redirect(facilityEndDateRedirectUri);
+    if (isUsingFacilityEndDate === undefined) {
```

Comment:
could this not be if (!isUsingFacilityEndDate)

Reply:
I agree this is a little odd, could we do this instead @MarRobSoftwire? Reversing the condition, also simplifies the conditional URLs. Something like this


```js
if (isUsingFacilityEndDate) {
  return res.redirect(facilityEndDateRedirectUri);
}

// is undefined
return res.redirect(bankReviewDateRedirectUri);